awx/main/analytics/collectors.py

```diff
@@ -309,6 +309,6 @@ def _copy_table(table, query, path):
                                  main_unifiedjobtemplate.status 
                                  FROM main_unifiedjobtemplate, django_content_type
                                  WHERE main_unifiedjobtemplate.polymorphic_ctype_id = django_content_type.id
-                                 ORDER BY main_unifiedjobtemplate.id ASC) TO STDOUT WITH CSV HEADER'''.format(since.strftime("'%Y-%m-%d %H:%M:%S'"))    
+                                 ORDER BY main_unifiedjobtemplate.id ASC) TO STDOUT WITH CSV HEADER'''
```

Comment:
@benthomasson is aware of this.  He said it's better we don't filter Unified Job Templates by the last collection time anyways.

Reply:
Ouch, this is actually a bug.  Is this something that needs backporting?

Reply:
Nope. Functionally this changes nothing.  @ryanpetrello I talked this through with @benthomasson this morning, he is aware of the fact that this doesn't filter by time and instead sends all unified job templates.  He said it is actually better that way.